qa/rpc-tests/p2p-segwit.py

```diff
@@ -302,12 +302,12 @@ def test_unnecessary_witness_before_segwit_activation(self):
         sync_blocks(self.nodes)
 
         # We'll add an unnecessary witness to this transaction that would cause
-        # it to be too large according to IsStandard.
+        # it to be non-standard.
         tx3 = CTransaction()
         tx3.vin.append(CTxIn(COutPoint(tx2.sha256, 0), CScript([p2sh_program])))
         tx3.vout.append(CTxOut(tx2.vout[0].nValue-1000, scriptPubKey))
         tx3.wit.vtxinwit.append(CTxInWitness())
-        tx3.wit.vtxinwit[0].scriptWitness.stack = [b'a'*400000]
+        tx3.wit.vtxinwit[0].scriptWitness.stack = [b'a']
```

Comment:
Ah, this code could be commented better I think.  I did intend for the transaction to be too large for standardness rules, to ensure that we were checking for premature witness before doing the IsStandard check (that change was a recent bugfix).  Perhaps just improve the comment, rather than change the test itself?